### ISSUE TITLE
Lj / LJ  Nj / NJ  case-sensitivity issue

### ISSUE THREAD

**Issue by erinmclaughlin** [google/roboto#234]
This issue still appears to be unresolved.  https://github.com/google/roboto/issues/55

As I understand it, I am unable to run 'make' on my Mac because it isn't recognizing case-sensitivity in glyph names. 

`  File "/Users/erinmclaughlin/env/lib/python2.7/site-packages/ufo2ft/outlineOTF.py", line 895, in setupTable_CFF
    glyph = self.allGlyphs[glyphName]`
`KeyError: 'Lj'`

Lj and Nj are being confused with LJ and NJ.

Could a new naming scheme be adopted so that these glyphs won't require a crazy work-around?

Thanks,
Erin

**Comment by m4rc1e** [comment]
Hey Erin,

I'll see what I can do here. I've hit the same issue.

**Comment by moyogo** [comment]
BTW https://github.com/google/roboto/issues/55#issuecomment-264672901 says how one can build Roboto fonts on Mac.

**Comment by m4rc1e** [comment]
@moyogo thank you! I'm building from Ubuntu now.

I still think there shouldn't be a workaround for this.

**Comment by m4rc1e** [comment]
I can make the fonts perfectly on Ubuntu 14.04. I'll see what the differences are.

**Comment by davelab6** [comment]
@m4rc1e this seems important to fix before anything else :)

**Comment by m4rc1e** [comment]
@davelab6 if we use @anthrotype's suggestion, we'll need to test the fonts thoroughly, write some regression unit tests and cut a new release.

**Comment by davelab6** [comment]
Yes. Is the problem that on Mac OS X the file system is case insensitive,
and the robofab glif file naming scheme doesn't account for that correctly,
but the defcon glif naming scheme does?

**Comment by m4rc1e** [comment]
@davelab6 I've had this problem with other chains involving Robofab. I can confirm that more modern build tools such as [Fontmake](https://github.com/googlei18n/fontmake) don't suffer with this issue.
